pkg/resources/statefulsets/minio-statefulset.go

```diff
@@ -748,11 +749,26 @@ func NewPool(t *miniov2.Tenant, wsSecret *v1.Secret, skipEnvVars map[string][]by
 		podVolumes = append(podVolumes, t.Spec.SideCars.Volumes...)
 	}
 
+	moreServers := pool.Servers > 4
+	unavailable := intstr.FromInt(1)
```

Comment:
sweet, but wouldn't this cause quorum lost with low EC? we could also "validate" that

Reply:
well the idea is for most scenarios, anyone setting EC:1 on a 5 node setup is already running on a thin blade!

Reply:
however I wanted to validate this based on the storage-class or some parity setting but that is not possible since it can be set later as well.